SurveyApp/Modules/Login/LoginPresenter.swift

```diff
@@ -32,11 +32,29 @@ extension LoginPresenter: LoginViewOutput {
     func viewDidLoad() {
         view?.configure()
     }
+    
+    func didPressLogin(email: String, password: String) {
+        if email.isEmpty || password.isEmpty { return }
```

Comment:
You may want to use `guard` here, for [the reason](https://www.hackingwithswift.com/quick-start/understanding-swift/when-to-use-guard-let-rather-than-if-let) 😁